kadmin/kadm_conn.c

```diff
@@ -246,6 +246,11 @@ start_server(krb5_context contextp, const char *port_str)
 	hints.ai_flags    = AI_PASSIVE;
 	hints.ai_socktype = SOCK_STREAM;
 
+	if (krb5_config_get_bool(context, NULL, "libdefaults", "block_dns",
+		NULL)) {
+	    hints.ai_flags &= ~AI_CANONNAME;
```

Comment:
There is no `getaddrinfo()` flag for disabling DNS lookups.  Is this change specifically for NetBSD?